### PR TITLE
feat: allow to change all measurements view at once

### PR DESCRIPTION
**Pull request by Sebastien-Ahkrin**
close: https://github.com/zakodium-oss/analysis-ui-components/issues/391

### CODE REVIEW

File: src/app-data/state/producers/measurements.ts [hunk 1]
```diff
@@ -108,6 +108,15 @@ export const selectMeasurementKind: AppStateProducer<
 export const setMeasurementVisibility: AppStateProducer<
   'SET_MEASUREMENT_VISIBILITY'
 > = (draft, action) => {
+  if (action.payload.id === null) {
+    for (const id of Object.keys(draft.view.measurements)) {
```
Comment (targos) [inline src/app-data/state/producers/measurements.ts:112]: You should only update the selected kind, like with "select all"

Comment (Sebastien-Ahkrin) [reply src/app-data/state/producers/measurements.ts]: do i have to update, only the selected items ?

Comment (targos) [reply src/app-data/state/producers/measurements.ts]: yes

Comment (Sebastien-Ahkrin) [reply src/app-data/state/producers/measurements.ts]: okay, so i will mark this conversation as resolved :)

Comment (targos) [reply src/app-data/state/producers/measurements.ts]: Actually, we discussed last week and the issue was wrong (I just updated it)
There should be two separate buttons with the eyes (to show and hide). Click on the buttons should only apply to the selected measurements.

For example, with this selection:
image

After the click to the open eye, it should be like this:
image

File: src/app-data/state/producers/measurements.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export const setMeasurementVisibility: AppStateProducer<
 };
 
 export const setAllMeasurementVisibility: AppStateProducer<
```
Comment (targos) [inline src/app-data/state/producers/measurements.ts:131]: rename this too

**Comment by targos** [comment]
Layout is broken: 
image

**Pull request merged**